Utilities/bootstrap

```diff
@@ -289,6 +291,9 @@ def test(args):
     # Test Yams.
     call_swiftpm(args, cmd, args.yams_source_dir)
 
+    # Test swift-argument-parser.
```

Comment:
I backed away from testing Yams and SwiftDriver, and I think this should be removed as well. Let's just test SwiftPM here.